manifests/internal-router/internal-router.cm.yaml

```diff
@@ -16,6 +16,7 @@ data:
         proxy_http_version 1.1;
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection "upgrade";
+        chunked_transfer_encoding off;
```

Comment:
is it still needed when we using version 1.22?
is so, how come that it's works out of the box up until now? maybe we are using very old version of nginx?